examples/purchase-tester/src/main/res/layout/fragment_overview.xml

```diff
@@ -12,207 +12,221 @@
         <import type="android.view.View" />
     </data>
 
-    <ScrollView
+    <FrameLayout
```

Comment:
Almost all the changes in this file are because I moved the `ScrollView` inside the `FrameLayout`. Nothing changed inside that container. The only real change is adding the button to navigate to the logs at the bottom of these changes.